app/Services/PushService.php

```diff
@@ -18,8 +18,12 @@ public function __construct()
         $this->snsClient = App::make('CustomSnsClient');
     }
 
-    public function fetchEndpointArn(string $os, string $pushToken): string
+    public function fetchEndpointArn(string $os, string $pushToken): ?string
     {
+        if (! config('app.sns.enabled')) {
+            return null;
+        }
```

Comment:
We should likely remove the SNS integration entirely as it's only useful for mobile apps, but for now this will do.